### PR TITLE
Review dependencies

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
Some changes extracted from #4624, so it's easier for you to review.

### CODE REVIEW

**Comment by deivid-rodriguez** [comment]
Ruby 2.4.0 failed as expected. I added what I think it's should be our build matrix for 2.4.0 in https://github.com/activeadmin/activeadmin/pull/4754/commits/47d9a511749e1e4b7c9ceac8f5172ed82bd89307. Thoughts?

**Comment by deivid-rodriguez** [comment]
Ready! Anyone?

File: spec/unit/page_spec.rb [hunk 1]
```diff
@@ -34,7 +34,11 @@ def config(options = {})
       end
 
       it "returns the singular, lowercase name" do
-        expect(config.resource_name.singular).to eq "chocolate i lØve you!"
+        if RUBY_VERSION >= '2.4.0'
```
Comment (Paxa) [inline spec/unit/page_spec.rb]: Both lines are same, isn't it?

Comment (deivid-rodriguez) [inline spec/unit/page_spec.rb]: No, case is different!

Comment (timoschilling) [inline spec/unit/page_spec.rb]: The `Ø` is lowercase in 2.4 and uppercase in others.

**Pull request merged**